workers/unity/Packages/com.improbable.gdk.core/View/View.cs

```diff
@@ -136,5 +122,26 @@ public bool IsAuthoritative(EntityId entityId, uint componentId)
 
             return componentIdToViewStorage[componentId].GetAuthority(entityId.Id) == Authority.Authoritative;
         }
+
+        internal void ApplyDiff(ViewDiff diff)
+        {
+            var entitiesAdded = diff.GetEntitiesAdded();
+            foreach (var entity in entitiesAdded)
+            {
+                entities.Add(entity);
+            }
+
+            var entitiesRemoved = diff.GetEntitiesRemoved();
+            foreach (var entity in entitiesRemoved)
+            {
+                entities.Remove(entity);
+            }
+
+            foreach (var storage in viewStorages)
+            {
+                // Resolve this with an actual diff!
```

Comment:
Nit: No need for the !

Reply:
I just copied this over.

Reply:
@jamiebrynes7 what is this comment for

Reply:
Leftover comment that I forgot to remove. Can remove